features/samples.feature

```diff
@@ -183,7 +191,13 @@ Feature: Basic smell detection
     Then the exit status indicates smells
     And it reports:
     """
-    redcloth.rb -- 95 warnings:
+    redcloth.rb -- 101 warnings:
+      RedCloth declares the writeable attribute filter_html (Attribute)
+      RedCloth declares the writeable attribute filter_styles (Attribute)
+      RedCloth declares the writeable attribute hard_breaks (Attribute)
+      RedCloth declares the writeable attribute lite_mode (Attribute)
+      RedCloth declares the writeable attribute no_span_caps (Attribute)
+      RedCloth declares the writeable attribute rules (Attribute)
```

Comment:
I think we should get rid of this whole redcloth/inline.rb thingie, but let's discuss this in my upcoming PR